micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -597,7 +610,9 @@ public More more() {
         return (M) m;
     }
 
-    private Meter getOrCreateMeter(HistogramConfig config, BiFunction<Id, HistogramConfig, Meter> builder, Id mappedId) {
+    private Meter getOrCreateMeter(@Nullable HistogramConfig config,
+                                   BiFunction<Id, /*Nullable Generic*/ HistogramConfig, Meter> builder,
```

Comment:
I'm not certain how to annotate the generic to indicate that `HistogramConfig` could be null. Perhaps we shouldn't allow it to be `null` ever?